rails/rails/action_view.yml

```diff
@@ -4,37 +4,37 @@
     distance_in_words:
       half_a_minute: "half a minute"
       less_than_x_seconds:
-        one:   "less than %{count} second"
+        one:   "less than 1 second"
```

Comment:
The decision to prefer `%{count}` was discussed in https://github.com/svenfuchs/rails-i18n/pull/993

Reply:
Correct, that is why #1088 leaves them as they are.
These changes are verbatim from https://github.com/rails/rails/blob/main/actionview/lib/action_view/locale/en.yml
This file does not belong to this project, it is only used as a reference.